### PR TITLE
Search feature

### PR DESCRIPTION
**Pull request by gathuku**
This will implement Search using algoria API on blogs page.
:Todo 
- Create an algolia community plan.
- Add indexes with name(zegetech.com)
- Add api keys 
Once setup complete run indexing 
```
ALGOLIA_API_KEY='your_admin_api_key' bundle exec jekyll algolia
``` 
[Reference](https://github.com/algolia/jekyll-algolia)

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -2356,8 +2356,10 @@ body.no-header-wrapper {
     -moz-background-clip: padding;
     -webkit-background-clip: padding-box;
     background-clip: padding-box;
-    width: 100%;
-    padding: 0;
+    /* width: 100%; */
+    min-width: 90% !important;
+    padding-bottom: 90px;
```
Comment (kgathi2) [inline assets/css/styles.css]: Please remove this and see if it's still ok. Should be. Seems quite big change and uneccessary. In as much as possible don't edit the styles already there

File: assets/css/styles.css [hunk 2]
```diff
@@ -2561,3 +2563,100 @@ code.highlighter-rouge{
...
+
+}
+.scrollable-menu {
+    height: auto;
+    max-height: 500px;
+    overflow-x: hidden;
+}
+.dropdown-menu:before{
+  position: fixed !important;
+}
```
Comment (kgathi2) [inline assets/css/styles.css]: I am not conformtable with overidding existing classes.

Comment (gathuku) [reply assets/css/styles.css]: Can i create a new class for the styles? For the dropdown

Comment (gathuku) [reply assets/css/styles.css]: styled the dropdown using an id

Comment (kgathi2) [reply assets/css/styles.css]: you choice of variable name is ambiguous. Call it something meaningful besides #theDropDown. Also check the classes you are overriding. Do a search in the file for that class and edit there.

**Pull request merged**